orchestra/todos/views.py

```diff
@@ -37,13 +40,35 @@ def update_todos_from_todolist_template(request):
         raise BadRequest('TodoList Template not found for the given slug.')
 
 
+@api_endpoint(methods=['GET'],
+              permissions=(permissions.IsAuthenticated,
+                           IsAssociatedWithProject),
+              logger=logger)
+def recommendations(request):
```

Comment:
recommendations is a loaded term for us---call them previous_todo_qas?

Reply:
Ok.